src/LaravelDataTables/Filters/SelectMultipleFilter.php

```diff
@@ -13,6 +14,22 @@ public function getSettings(): array
         ]);
     }
 
+    public function buildHtml(): string
+    {
+        return $this->getTemplate();
+    }
+
+    public function setTemplate($htmlTemplate): static
+    {
+        $this->template = $htmlTemplate;
+        return $this;
+    }
+
+    public function getTemplate(): string
```

Comment:
Why do we need a local $template for this class?

Reply:
To read the template if we want to just change some details on it, and then set the result back on the filter with setTemplate. The "template" is actually build at runtime calling buildHtml but the defaults are fine as they are.

Reply:
@Leonardo-Atalla can you link me to an example of it's usage in Infiny please?

Reply:
@danwithams-epsilon sure https://github.com/EpsilonTelecommunications/cloud-connect/blob/90e64d360cdcc9d0dd3b109e11eb453b59fecbeb/app/DataTables/DocumentRepository/CustomerDocumentsDataTable.php#L138

Reply:
@Leonardo-Atalla but isn't this what we have fixed by making its own filter Class? Surely it can have its own template in the package and not have to do the class replacements externally? That was one of the main points of me asking to make a new filter class for Multi-selects

Reply:
@danwithams-epsilon partially, we added an ad hoc class to set a couple of methods and properties on it to not make it necessary to extend it externally, yes, it can have its own template in the package, but that would be static then and we would need to agree on a default. I am ok with that, to clarify, all I wanted to be able to do was to make the multi filter wider extending on 12 bootstrap columns instead of 4. So if we agree on this or some defaults or a way to pass css classes to the container of the filter, I can remove the added template logic and declare default output in the class buildHtml (which would look very similar to the SelectFilter one with mentioned changes in width)

Reply:
resolved with a cssCol property in baseFilter class